microsoft-edge/progressive-web-apps-chromium/how-to/service-workers.md

```diff
@@ -275,12 +278,16 @@ It's helpful to know when a network connection is available, in order to synchro
 
 Use the following options to test for network connectivity:
 
+
+<!-- ------------------------------ -->
 #### navigator.onLine
 
 The `navigator.onLine` property is a boolean that lets you know the current status of the network. If the value is `true`, the user is online; otherwise, the user is offline.
 
 To learn more, see [navigator.onLine](https://developer.mozilla.org/docs/Web/API/NavigatorOnLine) on MDN.
 
+
+<!-- ------------------------------ -->
 #### Online and Offline Events
 
 You can take action when your network connectivity changes.  You can listen and take action in response to network events.  The events are available on the `window`, `document`, and `document.body` elements, as shown below:
```

Comment:
changed 300 from:
console.log("Oh no, you lost your network connection.");
to:
console.log("Network connection lost!");

orig was inconsistent, lacking !
new is lowkey / smooth/ consistent